pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -108,10 +108,18 @@ func (c *Cluster) Install(ctx context.Context) error {
 
 	configPath := c.GetWorkdirPath(runtime.ConfigName)
 
+	kwokControllerComponentPatches := runtime.GetComponentPatches(config, "kwok-controller")
+
 	etcdComponentPatches := runtime.GetComponentPatches(config, "etcd")
 	kubeApiserverComponentPatches := runtime.GetComponentPatches(config, "kube-apiserver")
 	kubeSchedulerComponentPatches := runtime.GetComponentPatches(config, "kube-scheduler")
 	kubeControllerManagerComponentPatches := runtime.GetComponentPatches(config, "kube-controller-manager")
+	extraLogVolumes, err := runtime.GetLogVolumes(ctx)
+	if err != nil {
+		return err
+	}
+	kwokControllerExtraVolumes := kwokControllerComponentPatches.ExtraVolumes
```

Comment:
Is there a reason why `KwokControllerComponentPatches.ExtraVolumes` are not mounted on kind? I wanted to merge them but as soon as I did, this workflow started failing: https://github.com/kubernetes-sigs/kwok/actions/runs/4631661137/jobs/8194800586?pr=417

It is failing on my machine, saying that `/tmp` mount is used multiple times (we used /tmp on all mounts)

If we should not include it, I'll fix this no problem, but I thought we are going to need extra mounts for the kwok controller as well

Reply:
We can probably do a PR to allow relative paths that will fix this one and extend the extra volume mounts? I can do it here but I'm afraid it is going to be too much for one PR (It is already huge)